### DISCUSSION TITLE
Content Preview is not showing after content upload

### DISCUSSION THREAD

**Discussion by devaj04** [Sunbird-Ed/Community#479] [Bugs] [answered]
Content Preview is not showing after content upload
contentID: do_31378807165662003212263

URL - https://obj.diksha.gov.in/ntp-content-production/content/assets/do_31378807165662003212263/busra-1-0224.pdf?45
if remove ?45 in the URL then PDF is showing

I have attached one mail screenshot for your reference

Please find the following Jira ticket for your reference:
https://project-diksha.atlassian.net/browse/DU-541

<img width="784" alt="Screenshot 2023-05-03 171055" src="https://user-images.githubusercontent.com/125238862/235910423-55565263-8186-49ff-9397-fa52f39ef490.png">
<img width="785" alt="Screenshot 2023-05-03 171115" src="https://user-images.githubusercontent.com/125238862/235910445-28d1b620-f680-458f-a23e-573a24d333bf.png">
<img width="686" alt="Screenshot 2023-05-03 171131" src="https://user-images.githubusercontent.com/125238862/235910458-a13871b7-ccd2-4886-a4ff-53d1a5ce2280.png">
<img width="948" alt="Screenshot 2023-05-03 171406" src="https://user-images.githubusercontent.com/125238862/235910472-70084a07-3b90-4a62-b8e5-b78f86efcbcd.png">
<img width="958" alt="Screenshot 2023-05-03 171425" src="https://user-images.githubusercontent.com/125238862/235910488-878ea70e-005e-46d2-ba97-c6c58f113921.png">
<img width="901" alt="Screenshot 2023-05-03 171203" src="https://user-images.githubusercontent.com/125238862/235910500-95a920e5-a41b-404b-8ea0-9a397874772d.png">

**Comment by Abhign** [comment]
@rajeevsathish @vinukumar-vs Can you check this ticket once.

CC: @Krishnaj20 @chitranshu-keshav

**Comment by vinukumar-vs** [comment, marked as answer]
As you mentioned in the description, The PDF file is loading when we remove the query parameter if the PDF file path. 
https://obj.diksha.gov.in/ntp-content-production/content/assets/do_31378807165662003212263/busra-1-0224.pdf?45

Why is it block the request when query parameter is passed? Can you please check with our Devops team to verify why the blob path is not accessible with query params?

This issue is similar to https://github.com/orgs/Sunbird-ED/discussions/478

Are you facing this issue for all different sizes of PDF contents or specific to large files? 

Can you also verify by uploading Video(MP4), EPUB, H5P contents to cross check this issue is present for different content types or specific to PDF only?

I would suggest to move this issue to [CoKreat BB discussions](https://github.com/orgs/Sunbird-coKreat/discussions) as they are the owning the "workspace" feature.

@rajeevsathish [@pallakartheekreddy](https://github.com/pallakartheekreddy)  [@arun1205](https://github.com/arun1205) @Krishnaj20

**Reply by devaj04** [reply]
@vinukumar-vs 
yes, the issue is coming for all types of content.
and issue created in https://github.com/orgs/Sunbird-coKreat/discussions
https://github.com/orgs/Sunbird-coKreat/discussions/79

**Reply by vinukumar-vs** [reply]
@devaj04 what is the reason for failure of accessing the file path with query parameter? 

The same was working with different service provider & its working with OCI for portal request but not for mobile request(though it is pointing to different proxy paths).
https://github.com/orgs/Sunbird-ED/discussions/478#discussioncomment-5789542

**Reply by devaj04** [reply]
content not found
<img width="696" alt="image" src="https://user-images.githubusercontent.com/125238862/236172384-0f88c6e3-b20d-48a3-921a-f771bd781211.png">

**Reply by chitranshu-keshav** [reply]
@devaj04 is this issue resolved? If yes then can you please update the resolution and mark this thread as answered. If not then please get back

**Comment by devaj04** [comment]
issue solved
